lib/kuby/plugins/rails_app/database.rb

```diff
@@ -51,7 +51,11 @@ def db_config
         end
 
         def db_configs
-          @db_configs ||= YAML.load(File.read(db_config_path))
+          @db_configs ||= begin
```

Comment:
Can we check `Psych::VERSION >= '4'` here instead?

Reply:
Sure
I will make the change.

Reply:
@camertron I have made the changes